doc/README-TACACS+

```diff
@@ -0,0 +1,45 @@
+Cracking TACACS+ hashes with JtR
+---------------------------------
+
+* Extract TACACS+ hashes from the .pcap file.
+
+  $ ../run/pcap2john.py target.pcap > hashes
+
+
+* Attack these hashes using JtR Jumbo.
+
+  $ ../run/john hashes
```

Comment:
Since pcap2john may output other types of hashes as well, we need `--format:tacacs-plus` here